taxi/urls.py

```diff
@@ -16,8 +22,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/manufacturer-create/",
```

Comment:
The URL path 'manufacturers/manufacturer-create/' should use hyphens instead of underscores. Consider changing it to 'manufacturers/manufacturer-create/'.